paper/paper.md

```diff
@@ -74,6 +74,8 @@ WordTokenizers.jl is currently being used by packages like [TextAnalysis.jl](htt
 
 ## Other similar softwares
 
+![Speed comparison of Tokenizers on IMDB Movie Review Dataset](speed_compare.png)
+
 There are various NLP libraries and toolkits written in other programming languages, available to Julia users for tokenization.
 [NLTK](https://github.com/nltk/nltk) and [Spacy](https://github.com/explosion/spaCy) packages provide with a variety of tokenizers, accessed to Julia users via `PyCall`.
```

Comment:
```suggestion
[NLTK](https://github.com/nltk/nltk) and [Spacy](https://github.com/explosion/spaCy) packages provide with a variety of tokenizers, accessed to Julia users via `PyCall`.
Shown above is a performance benchmark of using some of the WordTokenizers.jl tokenizers vs PyCalling the the default tokenizers from NLTK and Spacy.
This was evaluated on the ~127,000 sentences of the IMDB Movie Review Dataset.
It can be seen that the performance of WordTokenizers.jl is very strong.
```